support-frontend/assets/components/thankYou/guardianAdLite/whatNext.tsx

```diff
@@ -0,0 +1,37 @@
+import { css } from '@emotion/react';
+import { palette } from '@guardian/source/foundations';
+import BulletPointedList from '../utilityComponents/BulletPointedList';
+
+const bulletPointBlack = css`
+	& > li > div > svg > circle {
+		fill: ${palette.neutral[7]};
```

Comment:
Could we add this as a prop of our component instead of drilling through the css override?

Reply:
Hmmm, are there pro's/con's to either approach? Would have to add a lot of plumbing code to do this for just one single use?  If your sure or there is a further reason (readability?) to do am happy to change - what do you think?

Reply:
It's a much of a muchness, but putting it closer to the component might be easier to read in future. Just looking at this file there's it's not obvious why we need the selector `li > div > svg > circle`

Reply:
Ok readability is important, revised with a color property for the `BulletPointedList`. It moves the cssOverride closer to its containing `svg` component.

Reply:
It looks good! You could take it further and simply have `fill={colour}` and pass that all the way through

Reply:
Ah yes, done that too now.